### PR TITLE
Remove Moon Mode Toggle

### PR DESCRIPTION
**Pull request by Sonicrida**
### Changes

* Removes the moon mode toggle from the footer

### Screenshots

image

### CODE REVIEW

**Comment by nickmccurdy** [comment]
https://app.clubhouse.io/mycrypto/story/1315

File: common/v2/features/Layout/Footer/components/LogoBox.tsx [hunk 1]
```diff
@@ -11,16 +10,10 @@ export default function LogoBox() {
...
-        <section className="LogoBox-image-toggle">
-          <ThemeToggle />
-        </section>
       </section>
       <section className="LogoBox-text">
         <p>{translate('NEW_FOOTER_TEXT_13')}</p>
       </section>
-      <section className="LogoBox-toggle">
-        <ThemeToggle />
-      </section>
```
Comment (nickmccurdy) [inline common/v2/features/Layout/Footer/components/LogoBox.tsx]: Please remove SCSS class

File: common/v2/features/Layout/Footer/components/LogoBox.tsx [hunk 2]
```diff
@@ -11,16 +10,10 @@ export default function LogoBox() {
     <section className="LogoBox">
       <section className="LogoBox-image">
         <img src={logo} alt="Logo" />
-        <section className="LogoBox-image-toggle">
-          <ThemeToggle />
-        </section>
```
Comment (nickmccurdy) [inline common/v2/features/Layout/Footer/components/LogoBox.tsx]: Please remove SCSS class

File: common/v2/features/Layout/Footer/components/LogoBox.tsx (hunk 1 above)
Comment (Sonicrida) [reply common/v2/features/Layout/Footer/components/LogoBox.tsx]: Done

File: common/v2/features/Layout/Footer/components/LogoBox.tsx (hunk 2 above)
Comment (Sonicrida) [reply common/v2/features/Layout/Footer/components/LogoBox.tsx]: Done

**Pull request merged**